TreeTracker/UI/Views/Authentication/SignIn/SignInViewModel.swift

```diff
@@ -53,15 +53,19 @@ class SignInViewModel {
         self.loginType = loginType
     }
 
-    private var usernameValue: String = "" {
+    var usernameValue: String = "" {
         didSet {
             viewDelegate?.signInViewModel(self, didUpdateValidationState: usernameValid)
             viewDelegate?.signInViewModel(self, didUpdateLoginEnabled: loginEnabled)
         }
     }
 
     func updateUsername(username: String) {
-        usernameValue = username
+        if loginType == .phoneNumber {
```

Comment:
Maybe a switch on login type here would be better?

Reply:
I used switch case here. 

switch loginType {
        case .phoneNumber:
            usernameValue = Validation.cleanedPhoneNumber(phoneNumber: username)
        case .email:
            usernameValue = username
        }